consensus/ibft/ibft.go

```diff
@@ -77,7 +77,7 @@ type backendIBFT struct {
 
 	blockTime time.Duration // Minimum block generation time in seconds
 
-	customContractAddress types.Address // custom contract address used for on-chain interaction
+	customContractAddress string // custom contract address used for on-chain interaction
```

Comment:
There's a `types.StringToAddress(str string)` function that is useful to parse strings into Address. Actually you could do this above in hooks.go.